### DISCUSSION TITLE
Unity Not Opening

### DISCUSSION THREAD

**Discussion by dennisant** [empriselab/RCareWorld#76] [Help - Setting up / Configuring] [answered]
Hello!

I'm currently able to successfully go through the Testing Installation instructions well. Everything seems to be working properly (as shown in the screenshot below) besides the Unity window not showing up. I uninstalled and reinstalled open3d as well as brew install libomp, but still no luck. Any thoughts?
<img width="1728" alt="Screenshot 2024-07-12 at 2 38 49 PM" src="https://github.com/user-attachments/assets/9cb62da1-7a00-4c90-b9fd-5552f2c948b7">

**Comment by YoruCathy** [comment]
Hi @dennisant, are you using a Macbook? As mentioned in the instructions, sometimes it doesn't work with Macbooks. We highly recommend switching to the Ubuntu system.

**Reply by dennisant** [reply]
I am using a Macbook. Any recommendations on the best way to go about using Ubuntu on a Macbook?

**Reply by YoruCathy** [reply, marked as answer]
I'd suggest seeing if you can team up with someone with a Ubuntu computer instead of figuring out how to use Ubuntu on a Macbook.

**Reply by dennisant** [reply]
Unfortunately that may not be an option for me. Do you think running Ubuntu inside a virtual machine would work?

**Reply by YoruCathy** [reply]
We haven't tried that before so we can not guarantee that works, but that might be an option.
